packages/notary/actors/notary_actor.py

```diff
@@ -11,10 +11,17 @@ class notary_actor(JSBASE):
     def start_page(self):
         return "welcome to https://github.com/threefoldtech/home/issues/135"
 
-    def register(self, robot_id, key, content, content_signature, schema_out):
+    def bcdb_get(self):
+        zdb_cl = j.clients.zdb.start_test_instance()
```

Comment:
You should not start bcdb every time you do need to get an client instance to it.
BCDB should be started when gedis starts, and the you just need to have one client instance to be used here in the gedis handlers. 
Same remark for the line 18, you should not always add the models again.